newIDE/app/src/UI/TextField.js

```diff
@@ -249,6 +253,14 @@ export default class TextField extends React.Component<Props, {||}> {
                 min: props.min,
                 step: props.step,
               },
+              // Input adornment:
+              endAdornment: props.endAdornment ? (
```

Comment:
interesting I didn't know adornment was a thing 👍

Reply:
I remembered having seen that in the material-ui docs, turns out to be a good fit here! :)